src/elements/funnel-progress/src/stories.tsx

```diff
@@ -0,0 +1,46 @@
+/** @jsx jsx */
+import * as React from 'react'
+import { jsx } from '@emotion/core'
+import { number, select } from '@storybook/addon-knobs'
+
+import FunnelProgress from './'
+
+export default {
+  title: 'Elements|Funnel Progress'
+}
+
+const progressKnob = (name: string) =>
+  number(name, 0.25, {
+    range: true,
+    min: 0,
+    max: 1,
+    step: 0.05
+  })
+
+export const Example = () => {
+  const phase = select('phase', ['Current plan', 'Results', 'Apply'], 'Results')
+  return (
+    <FunnelProgress
+      phases={[
+        {
+          title: 'Current plan',
+          open: phase === 'Current plan',
+          complete: phase !== 'Current plan',
+          progress: progressKnob('Current plan')
+        },
+        {
+          title: 'Results',
+          open: phase === 'Results',
+          complete: phase !== 'Current plan' && phase !== 'Results',
+          progress: progressKnob('Results')
+        },
+        {
+          title: 'Apply',
+          open: phase === 'Apply',
+          complete: false,
+          progress: progressKnob('Apply')
+        }
+      ]}
+    />
```

Comment:
It would be nice to push the logic calculating whether a phase is open / complete down inside `<FunnelProgress>` so that the consumer doesn't have to do it themselves. Example API:

```typescript
<FunnelProgress
  currentPhase="Results"
  progress={0.6}
  phases={['Current plan', 'Results', 'Apply']}
/>
```

This also has the nice property of "making invalid states unrepresentable": there can only be one currently active phase.